detection/generate_transforms.py

```diff
@@ -191,6 +201,156 @@ def generate_detection_train_transform(
     return train_transforms
 
 
+def generate_detection_train_transform_using_coord(
+    image_key,
+    box_key,
+    label_key,
+    point_key,
+    gt_box_mode,
+    intensity_transform,
+    patch_size,
+    batch_size,
+    affine_lps_to_ras=False,
+    amp=True,
+):
+    """
+    Generate training transform for detection.
+
+    Args:
+        image_key: the key to represent images in the input json files
+        box_key: the key to represent boxes in the input json files
+        label_key: the key to represent box labels in the input json files
+        gt_box_mode: ground truth box mode in the input json files
+        intensity_transform: transform to scale image intensities,
+            usually ScaleIntensityRanged for CT images, and NormalizeIntensityd for MR images.
+        patch_size: cropped patch size for training
+        batch_size: number of cropped patches from each image
+        affine_lps_to_ras: Usually False.
+            Set True only when the original images were read by itkreader with affine_lps_to_ras=True
+        amp: whether to use half precision
+
+    Return:
+        training transform for detection
+    """
+    if amp:
+        compute_dtype = torch.float16
+    else:
+        compute_dtype = torch.float32
+
+    train_transforms = Compose(
```

Comment:
Hi @ericspod @atbenmurray @Nic-Ma and @mingxin-zheng, could you please help take a look whether this new transform pipeline make sense to you. I test it in this test [notebook](https://github.com/Project-MONAI/tutorials/blob/002153ab02e408c64056221e6cc5ce343134f1c6/detection/test-identical.ipynb), it achieve the same result as before.